db.sql

```diff
@@ -37,6 +37,43 @@ CREATE TABLE `conn_bundle` (
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8;
 /*!40101 SET character_set_client = @saved_cs_client */;
 
+--
+-- Dumping data for table `conn_bundle`
+--
+
+LOCK TABLES `conn_bundle` WRITE;
+/*!40000 ALTER TABLE `conn_bundle` DISABLE KEYS */;
+INSERT INTO `conn_bundle` VALUES ('b9967037-4db8-444c-a749-f1f47fc95c93',NULL,0,0,0,14,'test','placeholder','2022-04-21 13:24:36',NULL);
+/*!40000 ALTER TABLE `conn_bundle` ENABLE KEYS */;
+UNLOCK TABLES;
+
+--
+-- Table structure for table `keysets`
+--
+
+DROP TABLE IF EXISTS `keysets`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!40101 SET character_set_client = utf8 */;
+CREATE TABLE `keysets` (
+  `id` int(11) NOT NULL AUTO_INCREMENT,
+  `keysetid` varchar(50) NOT NULL,
+  `kskeys` longtext CHARACTER SET utf8mb4 COLLATE utf8mb4_bin DEFAULT NULL,
+  `polid` int(11) NOT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `key keyset_id` (`keysetid`)
```

Comment:
Why is it `key keyset_id` and not just `keyset_id`. It's also slightly confusing to me that we use `keysetid` without a space and `keyset_id` with a space. What is the difference?

Reply:
This is a sql constraint name (key keyset_id)  that I believe is required to make keysetid a unique key.  I have not been able to find a way to make the alias and the name the same.   I will keep looking, but I have not been able to find a clean way to make them the same.